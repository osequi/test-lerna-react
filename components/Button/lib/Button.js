import React from "react";

var Button = function Button() {
  return /*#__PURE__*/React.createElement("button", null, "Nuttonski");
};

export default Button;
//# sourceMappingURL=Button.js.map