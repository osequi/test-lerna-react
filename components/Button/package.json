{
  "name": "osequi-test-lerna-react-button",
  "version": "1.0.0",
  "description": "> TODO: description",
  "author": "osequi <cs@osequi.com>",
  "homepage": "https://github.com/osequi/test-lerna-react#readme",
  "license": "ISC",
  "main": "lib/Button.js",
  "directories": {
    "lib": "lib",
    "test": "__tests__"
  },
  "files": [
    "lib"
  ],
  "repository": {
    "type": "git",
    "url": "git+https://github.com/osequi/test-lerna-react.git"
  },
  "scripts": {
    "test": "echo \"Error: run tests from root\" && exit 1",
    "tsc": "tsc",
    "build": "babel src --out-dir lib --copy-files --source-maps --extensions \".ts,.tsx,.js,.jsx,.mjs\""
  },
  "bugs": {
    "url": "https://github.com/osequi/test-lerna-react/issues"
  },
  "devDependencies": {
    "@babel/cli": "^7.12.1",
    "@babel/core": "^7.12.3",
    "@babel/plugin-proposal-export-default-from": "^7.12.1",
    "@babel/plugin-proposal-export-namespace-from": "^7.12.1",
    "@babel/preset-env": "^7.9.6",
    "@babel/preset-react": "^7.12.5",
    "@babel/preset-typescript": "^7.12.1",
    "@types/react": "^16.9.56",
    "@types/react-test-renderer": "^16.9.3",
    "babel-core": "^6.26.3",
    "babel-jest": "^26.6.3",
    "babel-loader": "^8.2.1",
    "babel-preset-minify": "^0.5.1",
    "jest": "^26.6.3",
    "react": "^17.0.1",
    "react-dom": "^17.0.1",
    "react-test-renderer": "^17.0.1",
    "webpack": "^5.4.0",
    "webpack-cli": "^4.2.0"
  },
  "peerDependencies": {
    "react": "^17.0.1",
    "react-dom": "^17.0.1"
  }
}
