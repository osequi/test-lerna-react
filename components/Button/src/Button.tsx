import React from "react";

const Button = () => {
  return <button>Nuttonski</button>;
};

export default Button;
