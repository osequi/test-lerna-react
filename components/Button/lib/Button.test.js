import React from "react";
import renderer from "react-test-renderer";
import Button from "./Button";
it("Displays the button", function () {
  var component = renderer.create( /*#__PURE__*/React.createElement(Button, null));
  var tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=Button.test.js.map