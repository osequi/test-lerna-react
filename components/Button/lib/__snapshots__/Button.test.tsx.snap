// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays the button 1`] = `
<button>
  Nutton
</button>
`;
